bpf/bpf_lxc.c

```diff
@@ -1544,13 +1544,8 @@ ipv6_policy(struct __ctx_buff *ctx, int ifindex, __u32 src_label,
 				 &ct_state_new, *proxy_port > 0, false, ext_err);
 		if (IS_ERR(ret))
 			return ret;
-
-		/* NOTE: tuple has been invalidated after this */
 	}
 
-	if (!revalidate_data(ctx, &data, &data_end, &ip6))
```

Comment:
Nothing ever used or there was a usage and then a commit removed it but left the check? (Ok, if this is too boring for you to search for.)

Reply:
I only went back to v1.11, and even there it was dead code.